src/lib.rs

```diff
@@ -180,6 +182,12 @@ impl Build {
         self
     }
 
+    /// Set the minimum version required
+    pub fn min_version(&mut self, major: u8, minor: u8, micro: u8) -> &mut Self {
```

Comment:
Why `u8` here? I feel like it can only cause problems to not just use `usize` in this case

Reply:
sure, it can be changed that way.